dockerfiles/examples/nfs/README.md

```diff
@@ -0,0 +1,21 @@
+# HPCC NFS storage
+
+## Directory contents
+
+### hpcc-nfs/
+
+The hpcc-nfs helm chart will provision a NFS Server and optionally a default shared PVC using the default Storage Class.
+
+Example use:
+
+*helm install nfsstorage hpcc-nfs/*
+*helm install myhpcc hpcc/ --set global.image.version=XXX -f examples/local/values-nfs.yaml*
```

Comment:
Perhaps latest rather than XXX

Reply:
I'm guessing the * at start/end of these lines is to make the line appear bold when viewed in a md viewer, but it makes it very confusing if viewed as plain text. I suspect would be better without them (I notice they are not present in the localhost example)